src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="list__item">
+            <a
+              class="link is-active"
+              href="#"
```

Comment:
The `href` attribute is set to `#`, which might not be the intended behavior. If these links are placeholders, consider adding a comment to clarify. Otherwise, ensure they point to the correct URLs.